scenes/menu/preferences_window.gd

```diff
@@ -0,0 +1,25 @@
+extends PopupPanel
+
+@onready var timer_selection = $VBoxContainer/StartingTimersSelection
+@onready var best_of_selection = $VBoxContainer/BestOfSelection
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	timer_selection.select(timer_selection.get_item_index(GlobalSettings.CustomStartingTimer))
```

Comment:
if you want to persist settings, the select will probably need to be a function so you can also call Save afterwards

Reply:
Oh, I thought the persistent settings were being saved on quit or something in some code block I hadn't seen. it seems from what you're saying that I should be calling save after every persistent settings change - I'll do that and I can just delete it if that's wrong

Reply:
wait I see. I ought to be calling the functions in globalsettings, they save on change. will fix